draft-ietf-rats-corim.md

```diff
@@ -59,6 +59,7 @@ normative:
   I-D.ietf-sacm-coswid: coswid
   I-D.ietf-rats-architecture: rats-arch
   I-D.ietf-rats-eat: eat
+  I-D.draft-ietf-rats-concise-ta-stores: ta-store
```

Comment:
This is not the way I-D refs work.  If you want you can copy the ref I've left inline in one of my previous comments.

Reply:
This seems to be working and taking us to the IETF data tracker for concise TA Store?

Reply:
`draft-` is not needed